app/middlewares/validators/users.js

```diff
@@ -0,0 +1,65 @@
+const { users: errorMessages } = require('../../constants/errorMessages');
+
+const emailDefaultValidation = {
+  isEmail: {
+    bail: true,
+    errorMessage: 'Should be an email'
+  },
+  matches: {
+    options: /(\w|-|\.)+@(wolox)\.(\w|\.){2,15}$/i,
+    errorMessage: 'You may only use email addresses from wolox'
```

Comment:
You could use the constant `OnlyWoloxEmail` defined in yout constant file.

Reply:
done :)